javaagent-tooling/javaagent-tooling.gradle

```diff
@@ -31,8 +31,9 @@ dependencies {
   implementation(deps.opentelemetryKotlin) {
     // opentelemetry-extension-kotlin classes are injected into user classpath
     // where kotlin core libraries are already present
-    exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib-common'
-    exclude group: 'org.jetbrains.kotlinx', module: 'kotlinx-coroutines-core'
+    exclude group: 'org.jetbrains.kotlin'
+    exclude group: 'org.jetbrains.kotlinx'
+    exclude group: 'org.jetbrains', module: 'annotations'
```

Comment:
btw, I just remembered, I don't think these exclusions are needed anymore since https://github.com/open-telemetry/opentelemetry-java/pull/2793

Reply:
@mateuszrzeszutek Are you going to look into this or just go ahead and merge?

Reply:
I'll take a look at it 👀

Reply:
Javaagent jar without these exclusions is exactly the same, they can be removed 👍